highlight/README.md

```diff
@@ -1,15 +1,33 @@
 Chapel highlighting/formatting support
 ======================================
 
-This directory contains rudimentary support for code
-coloring/formatting in emacs and vim.  See the README files in the
-individual subdirectories for details on using the modes.
+This directory contains support for code coloring/formatting using
+various editors and highlighting tools.  The level of support and
+complete language coverage may vary between tools.
 
-It also contains support for and/or pointers to other syntax
-highlighting capabilities that Chapel supports.
+Highlighting for Editors
+------------------------
+
+* emacs/ : support for highlighting in emacs (see `emacs/README.rst`
+* geany/ : support for highlighting in geany (see `geany/geany-README.md`)
```

Comment:
Might be worth renaming `geany-README.md` to just `README.md` (that way if you're browsing the directory in github you'll see the readme.)